### PR TITLE
Remove the duplicate content from access-cluster.md

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref: #27547 
Removing the duplicate section from https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/
and adding the link to https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-services/ in place of that.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/access-cluster-services.md [hunk 1]
```diff
@@ -86,7 +86,18 @@ See [Access Clusters Using the Kubernetes API](/docs/tasks/administer-cluster/ac
 As mentioned above, you use the `kubectl cluster-info` command to retrieve the service's proxy URL. To create proxy URLs that include service endpoints, suffixes, and parameters, you append to the service's proxy URL:
 `http://`*`kubernetes_master_address`*`/api/v1/namespaces/`*`namespace_name`*`/services/`*`[https:]service_name[:port_name]`*`/proxy`
 
-If you haven't specified a name for your port, you don't have to specify *port_name* in the URL.
+If you haven't specified a name for your port, you don't have to specify *port_name* in the URL. You can also use the port number in place of the *port_name* for both named and unnamed ports.
+
+By default, the API server proxies to your service using http. To use https, prefix the service name with `https:`:
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/access-cluster-services.md:91]:
```suggestion
By default, the API server proxies to your service using HTTP. To use HTTPS, prefix the service name with `https:`:
```

File: content/en/docs/tasks/administer-cluster/access-cluster-services.md [hunk 2]
```diff
@@ -86,7 +86,18 @@ See [Access Clusters Using the Kubernetes API](/docs/tasks/administer-cluster/ac
...
+`http://`*`kubernetes_master_address`*`/api/v1/namespaces/`*`namespace_name`*`/services/`*`https:service_name:[port_name]`*`/proxy`
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/access-cluster-services.md:92]:
```suggestion
`http://<kubernetes_master_address>/api/v1/namespaces/<namespace_name>/services/<service_name>/proxy`
```

File: content/en/docs/tasks/administer-cluster/access-cluster-services.md [hunk 3]
```diff
@@ -86,7 +86,18 @@ See [Access Clusters Using the Kubernetes API](/docs/tasks/administer-cluster/ac
...
+
+The supported formats for the name segment of the URL are:
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/access-cluster-services.md:94]:
```suggestion
The supported formats for the `<service_name>` segment of the URL are:
```

Comment (vaibhav2107) [reply content/en/docs/tasks/administer-cluster/access-cluster-services.md:94]: Thanks @tengqm for reviewing and suggesting the changes, will update these.

**Comment by vaibhav2107** [comment]
Hi @tengqm ,
PTAL.

**Comment by vaibhav2107** [comment]
Hi @sftim @bradtopol @daminisatya, could you please take a look.

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 4]
```diff
@@ -214,115 +214,8 @@ In each case, the credentials of the pod are used to communicate securely with t
...
-
-### Using web browsers to access services running on the cluster
-
-You may be able to put an apiserver proxy url into the address bar of a browser. However:
-
-  - Web browsers cannot usually pass tokens, so you may need to use basic (password) auth.  Apiserver can be configured to accept basic auth,
-    but your cluster may not be configured to accept basic auth.
-  - Some web apps may not work, particularly those with client side javascript that construct urls in a
-    way that is unaware of the proxy path prefix.
+The previous section was about connecting the Kubernetes API server.  [This section](/docs/tasks/access-application-cluster/access-cluster/) is about
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md:217]: @vaibhav2107 . Should this section link to the other task page? Some edits:

The previous section describes how to connect to the Kubernetes API server. For information about connecting to other services running on a Kubernetes cluster, see [Access Cluster Services](/docs/tasks/administer-cluster/access-cluster-services/).

Comment (vaibhav2107) [reply content/en/docs/tasks/access-application-cluster/access-cluster.md:217]: Hi @kbhawkey, thanks for suggesting the change, will update them soon.
Could you help me little more about this ` Should this section link to the other task page?`

Comment (kbhawkey) [reply content/en/docs/tasks/access-application-cluster/access-cluster.md:217]: 👋 @vaibhav2107 .
I think it is okay. After removing the content, this section only contains the sentence with the link.

**Comment by vaibhav2107** [comment]
@kbhawkey, applied the changes you have suggested,,please have a look

**Pull request merged**